misc/dbt-materialize/README.md

```diff
@@ -44,11 +44,42 @@ incremental | NO | Use the `materializedview` materialization instead! dbt's inc
 dbt only supports a limited set of [materialization types](https://docs.getdbt.com/docs/building-a-dbt-project/building-models/materializations). To create other types of objects in Materialize
 via dbt, use the following Materialize-specific macros:
 
-Macro | Details
+Macro | Purpose
 ------|----------
 mz_generate_name(identifier) | Generates a fully-qualified name (including the database and schema) given an object name.
 mz_create_source(sql) | Given some [`CREATE SOURCE`](https://materialize.com/docs/sql/create-source/) sql statement, creates the source in the Materialize instance.
 mz_drop_source(name, if_exists, cascade) | [Drops the named source](https://materialize.com/docs/sql/drop-source/) in Materialize.
+mz_create_index(obj_name, default, idx_name, col_refs, with_options) | [Creates an index](https://materialize.com/docs/sql/create-index/) in Materialize.
+mz_drop_index(name, default, if_exists, cascade) | [Drops an index](https://materialize.com/docs/sql/drop-index/) in Materialize.
+
+### Additional macro details
+
+#### mz_drop_source(name, if_exists, cascade)
+
+Argument | Type | Detail
+---------|------|--------
+name     | string | Name of the source to drop
+if_exists | boolean | Does not return an error if the named source does not exist
+cascade | boolean | Drops the source and its dependent objects
```

Comment:
do these tables normally express their defaults/if they're required? 

---
Oh I see below that optional ones get marked optional. That seems fine if that's how everything else in the dbt ecosystem does it.